src/UrdFactory.sol

```diff
@@ -24,11 +25,11 @@ contract UrdFactory {
         bytes32 salt
     ) public returns (UniversalRewardsDistributor urd) {
```

Comment:
should we return the interface instead?

Reply:
Already discussed [here](https://github.com/morpho-org/universal-rewards-distributor/pull/46#discussion_r1332595931)

Reply:
and [here](https://github.com/morpho-org/universal-rewards-distributor/pull/46#discussion_r1332644032)